scss/pattern-lib.scss

```diff
@@ -1,12 +1,2 @@
-@import 'vanilla-framework/scss/vanilla';
-@include vanilla;
-
 @import 'docs-vanilla-theme/scss/theme';
-@include docs-content;
-@include docs-footer;
-@include docs-header;
-@include docs-nav;
-@include docs-notes;
-@include docs-rows;
-@include docs-wrapper;
-@include docs-toc;
+@include docs-vanilla-theme;
```

Comment:
Looks like your including the entire docs theme. That will include vanilla-framework too. You only want one version loaded?